float64 x_ref
float64 y_ref
float64 z_ref
float64 yaw_ref
---
bool update_status